### PR TITLE
Tests & utils  refactor

### PR DESCRIPTION
**Pull request by krogla**
- removed contract addresses from ENV, using `networks.json` instead
- add to utils getLastRpcBlock
- fixed smoke test
- refactoring & optimizations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,14 @@
     "deploy-testnet": "graph deploy --product hosted-service --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ lidofinance/lido-testnet --network goerli",
     "create-local": "env-cmd --use-shell graph create --node '$GRAPH_NODE' lidofinance/lido",
     "remove-local": "env-cmd --use-shell graph remove --node '$GRAPH_NODE' lidofinance/lido",
-    "deploy-local": "env-cmd --use-shell graph deploy --node '$GRAPH_NODE' --ipfs '$GRAPH_IPFS' lidofinance/lido --network mainnet -l v2.0.0",
+    "deploy-local": "env-cmd --use-shell graph deploy --node '$GRAPH_NODE' --ipfs '$GRAPH_IPFS' lidofinance/lido --network mainnet -l v2.0.1",
```
Comment (kolyasapphire) [inline package.json:22]: Do we need to hardcode a version here? From now on (after v2 release), that's probably not necessary

Comment (kolyasapphire) [reply package.json:22]: Same for `deploy-local-testnet`

Comment (kolyasapphire) [reply package.json:22]: Same question for other commands

File: tests/graphBalance.test.off.alt.js [hunk 2]
```diff
@@ -1,20 +1,18 @@
-import fs from 'fs'
 import { providers, Contract, constants } from 'ethers'
 import { jest } from '@jest/globals'
+import { getContractAddress } from './config.js'
+import bollingAbi from '../abis/Billing.json'
```
Comment (kolyasapphire) [inline tests/graphBalance.test.off.alt.js:4]: Typo: `bolling` -> `billing`

Comment (krogla) [reply tests/graphBalance.test.off.alt.js]: fixed

**Pull request merged**